### PR TITLE
testing GitHub workflow

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Added GitHub workflow.
Closes https://github.com/cennznet/cennzx-ui/issues/25

### CODE REVIEW

File: .github/workflows/push-master.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    name: Dummy
+    if: "contains(github.event.head_commit.message, '[CI Skip]')"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+    steps:
+    - name: dummy
+      run: |
+        echo "Dummy skip step"
```
Comment (holygits) [inline .github/workflows/push-master.yml:54]: why do we need this?

**Review by holygits — COMMENTED** [review]
lgtm.
There's an action you can add to cancel previous runs on push

**Review by eupston — COMMENTED** [review]
nice one looks good to me.

File: .github/workflows/push-master.yml (hunk 1 above)
Comment (KarishmaBothara) [reply .github/workflows/push-master.yml]: don't need have removed this..

**Pull request merged**